tests/utils.go

```diff
@@ -31,13 +31,15 @@ const (
 	waitTime            = 600 * time.Second
 	veleroCLI           = "velero"
 	forceBindAnnotation = "cdi.kubevirt.io/storage.bind.immediate.requested"
+	//fedoraUrl           = "docker://quay.io/kubevirt/fedora-with-test-tooling-container-disk"
+	alpineUrl = "docker://quay.io/kubevirtci/alpine-with-test-tooling-container-disk:2205291325-d8fc489"
 )
 
 func CreateVmWithGuestAgent(vmName string, storageClassName string) *kvv1.VirtualMachine {
```

Comment:
does this alping image has a guest agent?

Reply:
yes, that is why it worked here